packages/video_player/video_player_android/android/src/main/java/io/flutter/plugins/videoplayer/RemoteVideoAsset.java

```diff
@@ -59,6 +61,9 @@ MediaItem getMediaItem() {
 
   @Override
   MediaSource.Factory getMediaSourceFactory(Context context) {
+    if (assetUrl.toLowerCase(Locale.US).startsWith("rtsp://")) {
```

Comment:
Can you explain the need to do this:

```java
assetUrl.toLowerCase(Locale.US)
```

We aren't doing a similar massaging for asset or http/s URLs.

Reply:
Yeah, I think it's unnecessary. Removed